components/ItemCard.js

```diff
@@ -1,22 +1,89 @@
 import styled from 'styled-components';
 import DeleteIcon from '../public/assets/delete-icon.svg';
+import EditIcon from '../public/assets/edit.svg';
+import CheckIcon from '../public/assets/check.svg';
+import { useState } from 'react';
+
+export default function ItemCard({
+  name,
+  date,
+  quantity,
+  onRemoveItem,
+  id,
+  onEditItem,
+}) {
+  const [isEdited, setIsEdited] = useState(false);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState(false);
```
As you are not done with the edit  yet, i think this should be named isEdting :)